models/issues/comment_list.go

```diff
@@ -165,7 +168,7 @@ func (comments CommentList) loadOldMilestones(ctx context.Context) error {
 
 func (comments CommentList) getAssigneeIDs() []int64 {
 	return container.FilterSlice(comments, func(comment *Comment) (int64, bool) {
-		return comment.AssigneeID, comment.AssigneeID > 0
+		return comment.AssigneeID, comment.AssigneeID > 0 && comment.Assignee == nil
```

Comment:
I do not think it is right. At least, it doesn't match the name `getAssigneeIDs`

Reply:
eaf351ed5190db44824bb8d39ba3d5d15558894a revert this change.